vega_sim/null_service.py

```diff
@@ -487,6 +487,14 @@ def manage_vega_processes(
         log_name="node",
     )
 
+    for _ in range(500):
+        try:
+            requests.get(
+                f"http://localhost:{port_config[Ports.CORE_REST]}/blockchain/height"
+            ).raise_for_status()
```

Comment:
Should We add break if request is successful?

Reply:
We should yes :D